x-pack/plugins/ingest_manager/server/services/epm/packages/get.ts

```diff
@@ -31,17 +31,19 @@ export async function getPackages(
       Object.assign({}, item, { title: item.title || nameAsTitle(item.name) })
     );
   });
-  const searchObjects = registryItems.map(({ name, version }) => ({
+  // get the installed packages
+  const results = await savedObjectsClient.find<Installation>({
     type: PACKAGES_SAVED_OBJECT_TYPE,
-    id: `${name}-${version}`,
-  }));
-  const results = await savedObjectsClient.bulkGet<Installation>(searchObjects);
-  const savedObjects = results.saved_objects.filter(o => !o.error); // ignore errors for now
+  });
+  // filter out the base package
+  const savedObjectsVisible = results.saved_objects.filter(
+    o => o.attributes.name !== HiddenPackages.base
```

Comment:
I think we can use the internal property and avoid maintaining our own list https://github.com/elastic/kibana/blob/ba446f39002cc28c01964af67723a00194bd4176/x-pack/plugins/ingest_manager/common/types/models/epm.ts#L118